.github/workflows/sample-apps-with-v2-beta.yml

```diff
@@ -0,0 +1,24 @@
+name: Building sample apps with v2-beta action
+
+on:
+  push:
+    branches:
+      - master
+      - patch/**
```

Comment:
If we're looking to have a mechanism to test our action, I don't know if we necessarily need to kick off one of these builds each time someone pushes to `master` or a patch branch. Since this workflow is fixed to a specific version of our action, and that action's version is fixed to a specific image tag, wouldn't we essentially be running the same build each time unless a change was made in this workflow to update the version of our action used?

Reply:
yeah I agree and was also thinking about the same. Things could change in two places, one in the actions and other in the sample apps itself. I wanted a place to see the whole E2E flow and potentially point end users to an example of how things work. Ideally I would have a separate repo for samples which demo our github actions but I would like to avoid doing that for now.
Building on every merge to `master` might not be that bad since it is not as frequent as other private branches.
I was looking into using the `repository_dispatch` even too which will enable us to trigger an action from the actions repo to this repo whenever any changes happen. But this will not handle any changes to the sample app in this repo itself though.
Thoughts?

Reply:
That's true, we don't push to `master` that often, so we could do that as well, I just wasn't sure what kind of value we would get out of jobs that didn't change the workflow file since the results _should_ be the same (but maybe that's what the jobs check 😊). I think we should be fine with triggering the job on pushes to `master` and `actions/**`.